From 8a425e34050c70a5a1a047fc04af4acff4d797b9 Mon Sep 17 00:00:00 2001
From: Phil Staub <Phil.Staub@windriver.com>
Date: Fri, 24 Oct 2008 13:56:11 -0700
Subject: [PATCH] BSP files for CF CN38XX/CN58XX (cav_ebt5800) BSP

Configuration files for the Cavium Octeon CN38XX/CN58XX (cav_ebt5800) BSP.

Signed-off-by: Phil Staub <phil.staub@windriver.com>
---
 .../bsp/cav_ebt5800/cav_ebt5800-standard.scc       |    9 +
 .../kernel-cache/bsp/cav_ebt5800/cav_ebt5800.cfg   |  198 ++++++++++++++++++++
 .../kernel-cache/bsp/cav_ebt5800/cav_ebt5800.scc   |    1 +
 3 files changed, 208 insertions(+), 0 deletions(-)
 create mode 100644 wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800-standard.scc
 create mode 100644 wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.cfg
 create mode 100644 wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.scc

diff --git a/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800-standard.scc b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800-standard.scc
new file mode 100644
index 0000000..deb8b7c
--- /dev/null
+++ b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800-standard.scc
@@ -0,0 +1,9 @@
+define WRS_BOARD cav_ebt5800
+define WRS_KERNEL standard
+define WRS_ARCH mips64
+
+scc_leaf ktypes/standard cav_ebt5800-standard
+
+
+
+include cav_ebt5800.scc
diff --git a/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.cfg b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.cfg
new file mode 100644
index 0000000..90a594e
--- /dev/null
+++ b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.cfg
@@ -0,0 +1,198 @@
+..........................................................................
+.                                WARNING
+.
+. This file is a kernel configuration fragment, and not a full kernel
+. configuration file.  The final kernel configuration is made up of
+. an assembly of processed fragments, each of which is designed to
+. capture a specific part of the final configuration (e.g. platform
+. configuration, feature configuration, and board specific hardware
+. configuration).  For more information on kernel configuration, please
+. consult the product documentation.
+.
+..........................................................................
+CONFIG_MIPS=y
+CONFIG_CPU_CAVIUM_OCTEON=y
+CONFIG_CAVIUM_OCTEON_REFERENCE_BOARD=y
+# CONFIG_USE_RI_XI_PAGE_BITS is not set
+CONFIG_CAVIUM_OCTEON_CVMSEG_SIZE=2
+CONFIG_CAVIUM_OCTEON_USER_IO=y
+CONFIG_CAVIUM_OCTEON_USER_MEM=y
+CONFIG_SMP=y
+CONFIG_NR_CPUS=16
+CONFIG_CAVIUM_OCTEON_POW_ONLY_ETHERNET=m
+# CONFIG_FLATMEM_MANUAL is not set
+CONFIG_SPARSEMEM_MANUAL=y
+CONFIG_SPARSEMEM=y
+# CONFIG_TICK_ONESHOT is not set
+# CONFIG_HIGH_RES_TIMERS is not set
+CONFIG_GROUP_SCHED=y
+CONFIG_RELAY=y
+# CONFIG_PCSPKR_PLATFORM is not set
+CONFIG_SLAB=y
+# CONFIG_MARKERS is not set
+CONFIG_BLOCK_COMPAT=y
+CONFIG_PCI=y
+# CONFIG_BINFMT_MISC is not set
+CONFIG_MIPS32_COMPAT=y
+CONFIG_MIPS32_O32=y
+CONFIG_MIPS32_N32=y
+CONFIG_BINFMT_ELF32=y
+# CONFIG_XFRM_USER is not set
+# CONFIG_NET_KEY is not set
+# CONFIG_NET_IPIP is not set
+# CONFIG_NET_IPGRE is not set
+# CONFIG_INET_AH is not set
+# CONFIG_INET_ESP is not set
+# CONFIG_INET_IPCOMP is not set
+# CONFIG_INET_XFRM_TUNNEL is not set
+# CONFIG_INET_TUNNEL is not set
+# CONFIG_IPV6 is not set
+# CONFIG_NETFILTER_XTABLES is not set
+# CONFIG_IP_NF_QUEUE is not set
+# CONFIG_IP_NF_IPTABLES is not set
+# CONFIG_IP_NF_ARPTABLES is not set
+# CONFIG_IP_SCTP is not set
+# CONFIG_TIPC is not set
+# CONFIG_ATM is not set
+# CONFIG_BRIDGE is not set
+# CONFIG_VLAN_8021Q is not set
+# CONFIG_WAN_ROUTER is not set
+# CONFIG_NET_SCH_CBQ is not set
+# CONFIG_NET_SCH_HTB is not set
+# CONFIG_NET_SCH_HFSC is not set
+# CONFIG_NET_SCH_PRIO is not set
+# CONFIG_NET_SCH_RED is not set
+# CONFIG_NET_SCH_SFQ is not set
+# CONFIG_NET_SCH_TEQL is not set
+# CONFIG_NET_SCH_TBF is not set
+# CONFIG_NET_SCH_GRED is not set
+# CONFIG_NET_SCH_DSMARK is not set
+# CONFIG_NET_SCH_NETEM is not set
+# CONFIG_NET_CLS_TCINDEX is not set
+# CONFIG_NET_CLS_ROUTE4 is not set
+# CONFIG_NET_CLS_FW is not set
+# CONFIG_NET_CLS_U32 is not set
+# CONFIG_NET_CLS_RSVP is not set
+# CONFIG_NET_CLS_RSVP6 is not set
+# CONFIG_NET_PKTGEN is not set
+# CONFIG_CAN is not set
+CONFIG_MTD=y
+CONFIG_MTD_PARTITIONS=y
+# CONFIG_MTD_REDBOOT_PARTS is not set
+# CONFIG_MTD_CMDLINE_PARTS is not set
+# CONFIG_MTD_AR7_PARTS is not set
+CONFIG_MTD_CHAR=y
+CONFIG_MTD_BLKDEVS=y
+CONFIG_MTD_BLOCK=y
+CONFIG_MTD_CFI=y
+CONFIG_MTD_GEN_PROBE=y
+# CONFIG_MTD_CFI_ADV_OPTIONS is not set
+CONFIG_MTD_CFI_AMDSTD=y
+CONFIG_MTD_PHYSMAP=y
+CONFIG_MTD_CAVIUM_FLASH=y
+CONFIG_BLK_DEV_LOOP=y
+# CONFIG_BLK_DEV_CRYPTOLOOP is not set
+# CONFIG_BLK_DEV_NBD is not set
+CONFIG_IDE=y
+CONFIG_BLK_DEV_IDE=y
+CONFIG_BLK_DEV_IDEDISK=y
+CONFIG_IDE_GENERIC=y
+# CONFIG_DUMMY is not set
+# CONFIG_BONDING is not set
+# CONFIG_TUN is not set
+CONFIG_MII=y
+CONFIG_NET_PCI=y
+CONFIG_CAVIUM_ETHERNET=y
+# CONFIG_PPP is not set
+# CONFIG_SLIP is not set
+CONFIG_NETPOLL_TRAP=y
+# CONFIG_INPUT is not set
+# CONFIG_SERIO is not set
+# CONFIG_VT is not set
+CONFIG_SERIAL_8250=y
+CONFIG_SERIAL_8250_CONSOLE=y
+CONFIG_SERIAL_8250_NR_UARTS=2
+CONFIG_SERIAL_8250_RUNTIME_UARTS=2
+# CONFIG_HW_RANDOM is not set
+CONFIG_I2C=m
+CONFIG_I2C_CHARDEV=m
+CONFIG_I2C_OCTEON_TWSI=m
+CONFIG_SENSORS_EEPROM=m
+CONFIG_THERMAL=y
+# CONFIG_SOFT_WATCHDOG is not set
+
+CONFIG_RTC_LIB=y
+# CONFIG_REISERFS_FS is not set
+# CONFIG_OCFS2_FS is not set
+# CONFIG_AUTOFS_FS is not set
+# CONFIG_AUTOFS4_FS is not set
+CONFIG_FAT_FS=y
+CONFIG_VFAT_FS=y
+# CONFIG_CONFIGFS_FS is not set
+# CONFIG_MINIX_FS is not set
+# CONFIG_ROMFS_FS is not set
+# CONFIG_SMB_FS is not set
+# CONFIG_CIFS is not set
+CONFIG_NLS_CODEPAGE_437=y
+# CONFIG_NLS_CODEPAGE_737 is not set
+# CONFIG_NLS_CODEPAGE_775 is not set
+# CONFIG_NLS_CODEPAGE_850 is not set
+# CONFIG_NLS_CODEPAGE_852 is not set
+# CONFIG_NLS_CODEPAGE_855 is not set
+# CONFIG_NLS_CODEPAGE_857 is not set
+# CONFIG_NLS_CODEPAGE_860 is not set
+# CONFIG_NLS_CODEPAGE_861 is not set
+# CONFIG_NLS_CODEPAGE_862 is not set
+# CONFIG_NLS_CODEPAGE_863 is not set
+# CONFIG_NLS_CODEPAGE_864 is not set
+# CONFIG_NLS_CODEPAGE_865 is not set
+# CONFIG_NLS_CODEPAGE_866 is not set
+# CONFIG_NLS_CODEPAGE_869 is not set
+# CONFIG_NLS_CODEPAGE_936 is not set
+# CONFIG_NLS_CODEPAGE_950 is not set
+# CONFIG_NLS_CODEPAGE_932 is not set
+# CONFIG_NLS_CODEPAGE_949 is not set
+# CONFIG_NLS_CODEPAGE_874 is not set
+# CONFIG_NLS_ISO8859_8 is not set
+# CONFIG_NLS_CODEPAGE_1250 is not set
+# CONFIG_NLS_CODEPAGE_1251 is not set
+# CONFIG_NLS_ASCII is not set
+CONFIG_NLS_ISO8859_1=y
+# CONFIG_NLS_ISO8859_2 is not set
+# CONFIG_NLS_ISO8859_3 is not set
+# CONFIG_NLS_ISO8859_4 is not set
+# CONFIG_NLS_ISO8859_5 is not set
+# CONFIG_NLS_ISO8859_6 is not set
+# CONFIG_NLS_ISO8859_7 is not set
+# CONFIG_NLS_ISO8859_9 is not set
+# CONFIG_NLS_ISO8859_13 is not set
+# CONFIG_NLS_ISO8859_14 is not set
+# CONFIG_NLS_ISO8859_15 is not set
+# CONFIG_NLS_KOI8_R is not set
+# CONFIG_NLS_KOI8_U is not set
+# CONFIG_NLS_UTF8 is not set
+CONFIG_DEBUG_FS=y
+CONFIG_DEBUG_MEMORY_INIT=y
+CONFIG_SYSCTL_SYSCALL_CHECK=y
+# CONFIG_KGDB is not set
+# CONFIG_CRYPTO_NULL is not set
+# CONFIG_CRYPTO_TEST is not set
+# CONFIG_CRYPTO_CRC32C is not set
+# CONFIG_CRYPTO_MD4 is not set
+# CONFIG_CRYPTO_MICHAEL_MIC is not set
+# CONFIG_CRYPTO_SHA1 is not set
+# CONFIG_CRYPTO_SHA256 is not set
+# CONFIG_CRYPTO_SHA512 is not set
+# CONFIG_CRYPTO_WP512 is not set
+# CONFIG_CRYPTO_AES is not set
+# CONFIG_CRYPTO_ANUBIS is not set
+# CONFIG_CRYPTO_ARC4 is not set
+# CONFIG_CRYPTO_BLOWFISH is not set
+# CONFIG_CRYPTO_CAST5 is not set
+# CONFIG_CRYPTO_CAST6 is not set
+# CONFIG_CRYPTO_KHAZAD is not set
+# CONFIG_CRYPTO_SERPENT is not set
+# CONFIG_CRYPTO_TEA is not set
+# CONFIG_CRYPTO_TWOFISH is not set
+# CONFIG_CRYPTO_DEFLATE is not set
+# CONFIG_LIBCRC32C is not set
diff --git a/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.scc b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.scc
new file mode 100644
index 0000000..0e57734
--- /dev/null
+++ b/wrs/cfg/kernel-cache/bsp/cav_ebt5800/cav_ebt5800.scc
@@ -0,0 +1 @@
+kconf hardware cav_ebt5800.cfg
-- 
1.5.5.1

